server/channels/app/platform/shared_channel_notifier.go

```diff
@@ -34,21 +34,21 @@ func (ps *PlatformService) SharedChannelSyncHandler(event *model.WebSocketEvent)
 	if syncService == nil {
 		return
 	}
-	if isEligibleForEvents(syncService, event, sharedChannelEventsForSync) {
+	if isEligibleForEvents(syncService, event, model.ToStringArray(sharedChannelEventsForSync)) {
 		err := handleContentSync(ps, syncService, event)
 		if err != nil {
 			mlog.Warn(
 				err.Error(),
-				mlog.String("event", event.EventType()),
+				mlog.String("event", string(event.EventType())),
 				mlog.String("action", "content_sync"),
 			)
 		}
-	} else if isEligibleForEvents(syncService, event, sharedChannelEventsForInvitation) {
+	} else if isEligibleForEvents(syncService, event, model.ToStringArray(sharedChannelEventsForInvitation)) {
```

Comment:
Is the feasible to change the signature of `isEligibleForEvents` to accept a `[]model.WebsocketEventType`?

Reply:
oops, missed this review, will try and get back